Project.toml

```diff
@@ -1,5 +1,5 @@
 name = "KnetNLPModels"
-uuid = "f468eda6-eac5-11e8-05a5-ff9e497bcd19"
+uuid = "9601d21a-7b1e-4fc7-9dab-5c3f545997d5"
```

Comment:
this change is already in master.

Reply:
I don't understand why is this happening.
I though I made my branch from master, it does not make sense.

Reply:
you must not have the latest master locally